### PR TITLE
JBPM-5355 - Support case project

### PR DESCRIPTION
**Pull request by mswiderski**
@cristianonicolai @nmirasch could you please have a look?

### CODE REVIEW

File: jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/handlers/NewCaseProjectHandler.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        this.notification = notification;
+    }
+
+    private org.uberfire.client.callbacks.Callback<Project> configureCaseProjectCallback = new org.uberfire.client.callbacks.Callback<Project>() {
+        @Override
+        public void callback(Project project) {
+            caseProjectService.call(new RemoteCallback<Void>() {
+                @Override
+                public void callback(Void aVoid) {
+                    notification.fire( new NotificationEvent( "Case project " + project.getProjectName() + " successfully configured", NotificationEvent.NotificationType.SUCCESS ) );
```
Comment (cristianonicolai) [inline jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/handlers/NewCaseProjectHandler.java]: i18n please (below for the error message as well) :)

**Comment by cristianonicolai** [comment]
@mswiderski overall looks good just some minor comments. Apart from that I'm just wondering if we should move this into jbpm-console-ng-workbench-integration module instead as the case-mgmt modules are unlikely to be deployed as part of the workbench at this point. They will be part of the case custom app instead.

**Comment by mswiderski** [comment]
thanks @cristianonicolai will move it to workbench integration instead. Will add one more thing - to add custom work definitions like Milestone when project/package is created as well.

**Comment by mswiderski** [comment]
@cristianonicolai all updates and improvements are now in place and ready for review

File: jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-backend/src/main/resources/CaseWorkDefinitions.wid [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "icon" : "defaultservicenodeicon.png"
+  ],
+
+  [
+    "name" : "Milestone",
+    "parameters" : [
+      "Condition" : new StringDataType()
+    ],
+    "displayName" : "Milestone",
+    "icon" : "defaultemailicon.gif"
```
Comment (cristianonicolai) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-backend/src/main/resources/CaseWorkDefinitions.wid]: Perhaps defaultservicenodeicon.png would be more appropriate?

Comment (mswiderski) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-backend/src/main/resources/CaseWorkDefinitions.wid]: I actually used it first but this icon looks more appropriate even the name is saying something else :)

**Pull request merged**